src/reanimated2/animation/decay.ts

```diff
@@ -44,7 +44,14 @@ export function withDecay(
 ): Animation<DecayAnimation> {
   'worklet';
 
-  return defineAnimation<DecayAnimation>(0, () => {
+  const animationFunctionCall = {
+    functionName: 'withDecay',
+    // eslint-disable-next-line prefer-rest-params
+    functionArguments: [...arguments],
```

Comment:
Can you please add comment why we can't just pass `arguments` and we need to make a copy?

Reply:
In all 6 places? 😅 
The problem with is that `arguments` is not an array (and typescript also agrees that it's not an array) and we need an array:
```js
> function foo() {
... arguments.map(a => console.log(a))
... }
undefined
> foo()
Uncaught TypeError: arguments.map is not a function
    at foo (REPL23:2:11)
```